C-Plus-Plus/ds/Next_Larger_Element-tree_using_CPP.cpp

```diff
@@ -2,12 +2,12 @@
 i.e. find a node with a value just greater than n.
```

Comment:
Remove the entire Redundant file

Reply:
@siddharth25pandey i merged this file next larger element by mistake. its not related to this PR. this code has already been merged. is there any way we can delete it from this PR.

Reply:
@anshika272 Yup, you can do in two ways, first one through command line (terminal) & second way is removing the file manually from Github GUI. Another way is to create a Fresh new PR if earlier two ways are not easy for you then.

Reply:
okay ill try it